Dockerfile

```diff
@@ -28,9 +28,11 @@ RUN npm ci --omit=dev
 
 COPY --from=builder /usr/app/dist/ dist/
 
+RUN npm link .
+
 LABEL org.opencontainers.image.source="https://github.com/nmshd/connector"
 
 USER node
 
 ENTRYPOINT ["/sbin/tini", "--"]
-CMD ["node", "/usr/app/dist/index.js"]
+CMD ["nmshd-connector", "start"]
```

Comment:
we can make this better:

```
ENTRYPOINT ["/sbin/tini", "--", "node", "/usr/app/dist/index.js"]
CMD ["start"]
```

please test this properly as I am not really sure if the syntax is 100% correct

Reply:
> Ok, but this would not allow the user to connect to a container and run some commands manually

that's not how docker exec works.

Reply:
Ok, can you explain this? Normally I run docker ... exec sh|bash and there I have the CLI available as a command.

Reply:
But you will never need the exec in the container and then execute the connector again.